orienteer-pivottable/src/main/java/org/orienteer/pivottable/component/widget/BrowsePivotTableWidget.java

```diff
@@ -56,7 +58,7 @@ public BrowsePivotTableWidget(String id, IModel<OClass> model,
 	}
 
 	public String getUrl() {
-		String sql = getSql();
+		String sql = ((customSQL != null) && (!customSQL.isEmpty())) ? getCustomSQL() : getSql();
```

Comment:
Why do not use as previously method "getSql()" and have required logic within that method?
More over: there is cool method Strings.isEmpty() for checks like this.